lib/ui/common/network_image_from_future.dart

```diff
@@ -1,34 +1,54 @@
-import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 
+class _Constants {
+  static final placeholderAsset = 'assets/raw/placeholder.webp';
+  static final defaultBorderRadius = BorderRadius.all(Radius.circular(0));
+}
+
 class NetworkImageFromFuture extends StatelessWidget {
   final Future<String> futureUrl;
   final double height;
   final double width;
-  final BoxFit fit;
+  final BorderRadius imageBorderRadius;
 
-  NetworkImageFromFuture(this.futureUrl, {this.height, this.width, this.fit});
+  NetworkImageFromFuture(
```

Comment:
I wonder if we should go a step further and add a constructor that takes a ImageProvider (the FarmSmart one)

Reply:
we could call this ImageProviderView then

Reply:
done!